src/components/SidePanel/SidePanel.stories.mdx

```diff
@@ -0,0 +1,85 @@
+import { Meta, Story, ArgsTable, Canvas } from '@storybook/addon-docs/blocks'
+import { boolean, number, text, select } from '@storybook/addon-knobs'
+import SidePanel from './SidePanel'
+import SidePanelApp from './SidePanel.storiesHelpers'
+
+<Meta
+  title="Components/Overlay/SidePanel"
+  component={SidePanel}
+  parameters={{
+    design: {
+      type: 'figma',
+      url:
+        'https://www.figma.com/file/v8ReousbOpHN2g9n5n6StX/Standalone-Messages---2.0?node-id=342%3A2552',
+    },
+  }}
+/>
+
+# SidePanel
+
+Component that brings a Panel from either side of the screen, it includes a default Header and Footer (both can be turned off via their respective props if you need to fully customize the panel's contents). The footer contains a CTA button.
+
+This component is fully controlled via the `show` prop, meaning it doesn't hold any internal state about whether is open or not, it's in your hands to manage this. The panel will fire the close action (`onClose` callback prop) with the &times; close button and via pressing the <kbd>esc</kbd> key.
```

Comment:
I really like that! I mean, a lot of components in HSDS have some internal state as a duplication of provided props and in most of the cases it only adds complexity. Having it outside makes that this component is quite simple :)